### ISSUE TITLE
Adding the ability to display buttons under a post and a post translation button

### ISSUE THREAD

**Issue by KyKyDesign** [cheeaun/phanpy#575]
Hello, I would like the option to be added to enable the number of likes, boosts and comments to be displayed under the posts, this will also enable quick toot boost.

I would like there to be a "translate" button immediately on the post under the text.

**Comment by nycki93** [comment]
agreed, it feels weird that it takes so many clicks to boost a post.
